user-access-service/build.gradle

```diff
@@ -39,4 +40,5 @@ dependencies {
 
 test{
     environment "API_HOST", "api.dev.aws.nva.unit.no"
+    environment "USERS_AND_ROLES_TABLE", "UsersAndRolesTable"
```

Comment:
removed the mocked environments that all they were doing was to supply a table name